app/components/press-release/form.js

```diff
@@ -74,6 +75,27 @@ export default class PressReleaseFormComponent extends Component {
     sources.removeObject(source);
   }
 
+  @action
+  async addContactItems(newContactItems) {
+    const contactLists = await this.args.pressRelease.contactItems;
+    const publicationChannels = await this.args.pressRelease.publicationChannels;
+    contactLists.pushObjects(newContactItems);
+    publicationChannels.pushObject(this.mailingListPublicationChannel);
+    this.showContactItemModal = false;
+  }
+
+  @action
+  async removeContactItem(contactItem) {
+    const contactItems = await this.args.pressRelease.contactItems;
+    const contactLists = await this.args.pressRelease.contactItems;
```

Comment:
moet dit niet zijn
const contactLists = await this.args.pressRelease.contactLists;